igb/repl.go

```diff
@@ -38,172 +38,150 @@ const (
 	emojis = "😀😁😂🤣😃😄😅😆😉😊😋😎😍😘😗😙😚🙂🤗🤔😐😑😶🙄😏😮😪😴😌😛😜😝🤤🙃🤑😲😭😳🤧😇🤠🤡🤥🤓😈👿👹👺💀👻👽🤖💩😺😸😹😻😼😽"
 )
 
+// Igb holds internal states of Igb.
+type Igb struct {
+	sm        *fsm.FSM
+	rl        *readline.Instance
+	completer *readline.PrefixCompleter
+	line      string
+	cmds      []string
+	stack     int
+}
+
+// Ivm holds VM only for Igb.
+type Ivm struct {
```

Comment:
In the function (`createVM`) below you uppercased the `VM` word, why don't we also uppercase vm here? Normally in Golang it should be uppercased since it's an abbreviation, like JSON or HTML.